langchain/document_loaders/blob_loaders/schema.py

```diff
@@ -0,0 +1,156 @@
+"""Schema for Blobs and Blob Loaders.
+
+The goal is to facilitate decoupling of content loading from content parsing code.
+
+In addition, content loading code should provide a lazy loading interface by default.
+"""
+import contextlib
```

Comment:
Minor nit is we import `from __future__ import annotations` to resolve the forward references (so we don't have to return `"Blob"`)

Reply:
TIL -- maybe I knew this before, but definitely forgot if I did